git-big-picture

```diff
@@ -269,25 +271,6 @@ def debug(message):
         sys.stdout.write('debug:   %s\n' % message)
 
 
-def parse_variable_args(args):
-    """ Parse arguments and get repo_dir.
-
-    Parameters
-    ----------
-    args : list
-        arguments given on command line
-
-    Returns
-    -------
-    repo_dir : path
-        path to the repo_dir
-
-    """
-    if len(args) > 1:
-        barf('Too many arguments: %s' % args, EXIT_CODES["too_many_args"])
```

Comment:
This changes the output and error message in case too many arguments are supplied on the command line.

```
 💣 zsh» ./git-big-picture -C -c -v open . ../numba
usage: git-big-picture OPTIONS [REPOSITORY]
git-big-picture: error: unrecognized arguments: ../numba
```

I think the error message is harder too understand than `Too  many arguments`. Would it be possible to reinstate this functionality (and add some tests)? Or is that something now handled by `argparse` and hard to implement?

Reply:
I agree that `unrecognized arguments` is not cristal clear but I think the usage line above is and it's the same message for everything based on argparse. I can use `nargs='+'` and do custom handling of more than a single parameter. I'm just not sure if it's worth the extra code.

Reply:
@esc I have restored the "Too many arguments" logic for you now. if you change your mind, I have a backup of the previous commit.

Reply:
Sounds good, thanks.